src/proxy/index.js

```diff
@@ -23,6 +25,10 @@ proxyApp.use(bodyParser.raw(options));
 proxyApp.use('/', router);
 
 const start = async () => {
+   const plugins = config.getPlugins();
+   const pluginLoader = await createLoader(plugins);
+   await pluginLoader.load;
```

Comment:
This is a nitpick, but it feels weird for something async to be a property and not a method.

Reply:
Some of this logic can be simplified. We do need to block at certain points to ensure that the PluginLoader has parsed all the modules and added it to its list of plugin objects before proceeding with the rest of the app startup.

If this were a method, we would need to do some sort of `while { }` loop with a predefined timeout to "block" on completion. It's more straight forward to simply `await` on a Promise that will eventually resolve.

Open to suggestions to improve this section but this code is working well enough in a live environment so I'm inclined to retain it.